### DISCUSSION TITLE
Training times

### DISCUSSION THREAD

**Discussion by aprikyan** [ultralytics/yolov5#3408] [Q&A] [answered]
Hey!

I see the training took 2/4/6/8 days for YOLOv5s/m/l/x on a single V100. Could you also provide similar stats for YOLOv5-P6 models?

**Comment by glenn-jocher** [comment, marked as answer]
@aprikyan the P6 models are about the same speed (train and detect) as the P5 models at the same `--image-size`. 

The P6 models are capable of training at larger image sizes though, which independently takes longer.

**Reply by aprikyan** [reply]
Got it, thanks. What about the training duration of the 5 pretrained on 1280px P6 models, in particular?

**Reply by glenn-jocher** [reply]
@aprikyan --img 1280 training takes 4x longer than --img 640 training, all else equal.

**Reply by aprikyan** [reply]
Thanks
